tests/e2e/pages/CheckoutPageSFRA.js

```diff
@@ -88,7 +88,7 @@ export default class CheckoutPageSFRA {
 
     setEmail = async () => {
         await t
-            .typeText(this.checkoutPageUserEmailInput, 'test@adyenTest.com');
+            .typeText(this.checkoutPageUserEmailInput, 'test@adyenTest.com', {replace: true});
```

Comment:
Why is the `{replace: true}` needed?

Reply:
When you return from the redirect this field might already be filled in (just like the rest of the form).
If I leave this out it will type the email in a field which already contains the email address